common/src/main/java/whocraft/tardis_refined/client/renderer/RenderHelper.java

```diff
@@ -0,0 +1,55 @@
+package whocraft.tardis_refined.client.renderer;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.*;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.phys.AABB;
+import net.minecraft.world.phys.Vec3;
+import org.joml.Matrix4f;
+
+public class RenderHelper {
+
+    public static void renderFilledBox(PoseStack stack, VertexConsumer vertexConsumer, AABB box, float red, float green, float blue, float alpha, int combinedLightIn) {
```

Comment:
This code looks dead familiar, Regeneration? https://github.com/Suff99/Regeneration/blob/82a4b27b69dccbbcf9a53ac3131e3ea6ae806dac/common/src/main/java/mc/craig/software/regen/util/RenderHelp.java#L15

Regardless, looks good

Reply:
Regeneration -> Whocraft Mobs -> Refined